payments-ui-core/src/test/java/com/stripe/android/ui/core/forms/resources/LpmRepositoryTest.kt

```diff
@@ -250,9 +250,11 @@ class LpmRepositoryTest {
     }
 
     @Test
-    fun `Verify that us_bank_account is supported when financial connections sdk available`() {
+    fun `Verify that us_bank_account is not supported when is payment intent and financial connections sdk available`() {
```

Comment:
Is this a behavior change, I'd expect it to be available here.

Reply:
Sorry, the test name is wrong. Updating.